lib/format/vtt.ts

```diff
@@ -38,7 +38,7 @@ const parse = (content: string, options: ParseOptions) => {
     for (let i = 0; i < parts.length; i++) {
         // WebVTT data
         const regex =
-            /^([^\r\n]+\r?\n)?((?:\d{1,2}:)?\d{1,2}:\d{1,2}(?:[.,]\d{1,3})?)\s*-->\s*((?:\d{1,2}:)?\d{1,2}:\d{1,2}(?:[.,]\d{1,3})?)\s?.*\r?\n([\s\S]*)$/;
+            /^([^\r\n]+\r?\n)?((?:\d{1,2}:)?\d{1,2}:\d{1,2}(?:[.,]\d{1,3})?)\s*-->\s*((?:\d{1,2}:)?\d{1,2}:\d{1,2}(?:[.,]\d{1,3})?)[ ]*.*\r?\n([\s\S]*)$/;
```

Comment:
Perhaps instead of specifying only space, we can use `[^\S\r\n]`? In that way it should match `\s` without new lines.

This seems to be working for me: `^([^\r\n]+\r?\n)?((?:\d{1,2}:)?\d{1,2}:\d{1,2}(?:[.,]\d{1,3})?)\s*-->\s*((?:\d{1,2}:)?\d{1,2}:\d{1,2}(?:[.,]\d{1,3})?)[^\S\r\n]?.*\r?\n([\s\S]*)$`

Reply:
By the way, I'm thinking of refactoring all RegExps to use named capturing groups since it's easier to read, but maybe after merging this PR.

Reply:
That's a better approach.